deployment/build-and-stage.yaml

```diff
@@ -98,6 +98,15 @@ steps:
   args: ['push', '--all-tags', 'gcr.io/oss-vdb/alias-computation']
   waitFor: ['build-alias-computation', 'cloud-build-queue']
 
+- name: gcr.io/cloud-builders/docker
+  args: ['build', '-t', 'gcr.io/oss-vdb/staging-test:latest', '-t', 'gcr.io/oss-vdb/staging-test:$COMMIT_SHA', '.']
+  dir: 'docker/staging_test'
+  id: 'build-staging-test'
+  waitFor: ['build-worker']
+- name: gcr.io/cloud-builders/docker
+  args: ['push', '--all-tags', 'gcr.io/oss-vdb/staging-test']
```

Comment:
Since we don't intend to put this to production, thoughts on naming/pushing this to `gcr.io/oss-vdb-test`?
I believe this Cloud Run service account already has the permissions to push to there.

Reply:
yep, good idea. Just moved to `oss-vdb-test`